fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -31,8 +34,11 @@ class _Lagrangian:
         the estimator to fit in every iteration of :meth:`best_h` using a
         :meth:`fit` method with arguments `X`, `y`, and `sample_weight`
     constraints : fairlearn.reductions.Moment
-        Object describing the parity constraints. This provides the reweighting
+        Object describing the fairness constraints. This provides the reweighting
```

Comment:
Why this switch?

Reply:
Our fairness constraints are actually more generic than what I'd call "parity" constraints. For example, bounded group loss is not really a parity constraint. People are confused: https://stackoverflow.com/questions/66910107/can-you-use-fairlearn-for-non-parity-constraints-binned-monotonicity/
Also, we could say "the optimization constraints" vs "the optimization objective". What do you think?

Reply:
That's OK with me. It might be good to put the note about that in the PR description. I just wasn't expecting this to be the first change I saw.